tests/test_app.py

```
@@ -0,0 +1,102 @@
+
+# Import the app module components for testing
+# Note: We import functions directly to avoid module-level side effects
+from app import configure_logging
+
+
+class TestConfigureLogging:
+    """Tests for configure_logging function."""
+
+    @patch("app.setup_logging")
+    @patch("app.Path")
+    @patch.dict("os.environ", {"MAIN_DIR": "/tmp/test_main_dir"}, clear=False)
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_main_dir"

(S108)